README.md

```diff
@@ -195,6 +195,25 @@ job('downstreamJob') {
 }
 ```
 
+### Pipeline support
```

Comment:
actually, in the above dsl sample code there is no way to use the comment file.  do you know the syntax for this?

Reply:
Looks like https://github.com/jenkinsci/ghprb-plugin/pull/637 is needed for dsl.